domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgress.kt

```diff
@@ -70,6 +79,17 @@ internal class ExplorationProgress {
     }
   }
 
+  /**
+   * Updates the checkpointState to a new state depending upon the result of save operation for
+   * checkpoints.
+   *
+   * @param newCheckpointState is the latest checkpoint state that is returned upon
+   *        completion of the save operation for checkpoints either successfully or unsuccessfully.
```

Comment:
```suggestion
   * @param newCheckpointState is the latest checkpoint state that is returned upon
   *      completion of the save operation for checkpoints either successfully or unsuccessfully.
```

Continuation is +4 from the normal KDoc spacing.

Reply:
done

Reply:
done

Reply:
done